Big-Spender/readme.md

```diff
@@ -68,39 +75,59 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT SELECT ID, DESCRIPTION FROM SPENDS
+WHERE LOWER(DESCRIPTION) like '%fee%'
```

Comment:
not a blocker but: 
-remember to remove the insert 
- include the ; at the end

--also can try with using` ilike` as this will make it case insensitive ;)